### ISSUE TITLE
Document training hyper-parameters and recommended settings

### ISSUE THREAD

**Issue by thesfy** [ICSResearch/ACIU-Net#10]
For different tasks / configurations (e.g. different compression ratios, noise levels), a table of recommended hyperparameters (learning rate, batch size, number of epochs, scheduling, etc.) would be very helpful.
